pkg/asset/config/appliance_config.go

```diff
@@ -248,20 +255,33 @@ func (a *ApplianceConfig) validateConfig() field.ErrorList {
 }
 
 func (a *ApplianceConfig) validateImageRegistry() field.ErrorList {
+	allErrs := field.ErrorList{}
+
 	if a.Config.ImageRegistry == nil {
 		return nil
 	}
 
-	// TODO(MGMT-15163): add validation for a.Config.ImageRegistry.URI
+	if a.Config.ImageRegistry.URI != nil {
+		cmd := fmt.Sprintf(PodmanInspect, swag.StringValue(a.Config.ImageRegistry.URI))
+		logrus.Debugf("Running uri validation cmd: %s", cmd)
+		args := strings.Split(cmd, " ")
+		_, err := a.executer.Execute(args[0], args[1:]...)
```

Comment:
nit: we can have it in one line.
i.e. if _, err := a.executer.Execute(args[0], args[1:]...); err != nil...

Reply:
done